docs/developer-docs/latest/setup-deployment-guides/deployment/strapi-cloud.md

```diff
@@ -23,12 +23,13 @@ Onboarding of users will be done in batches. You will be notified via email when
 
 Before you can deploy your Strapi application on Strapi Cloud, you need to have the following prerequisites:
 
-* A [GitHub](https://github.com) account
-* GitHub repository for your Strapi application(s)
-* Strapi version 4.1+
+* Strapi version `4.4.x` or higher
+* Database: PostgreSQL, MySQL, or MariaDB
```

Comment:
Seems there was a mistake in our FAQ, I just updated it. 
Thus, can we please change this line to "Database: your project should be compatible with PostgreSQL. Usage of any external database is not supported""